Sky Atlas/Sky Atlas.html

```diff
@@ -24302,23 +24325,47 @@ <h1>Agent Scope Database</h1>
           </td>
           <td>Spark</td>
           <td>Core</td>
-          <td>The Genesis Supply will be minted to an account owned by SPK Company Ltd and operationally controlled by Operational GovOps on behalf of SPK Company Ltd.</td>
+          <td><p>The Genesis Supply was minted to an account owned by SPK Company Ltd. The address of the account on the Ethereum Mainnet is <a href="https://etherscan.io/address/0x6FE588FDCC6A34207485cc6e47673F59cCEDF92B">0x6FE588FDCC6A34207485cc6e47673F59cCEDF92B</a>.</p></td>
         </tr>
         <tr>
           <td>
             <dfn>Transfer Of Tokens To Sky</dfn>
           </td>
           <td>Spark</td>
           <td>Core</td>
-          <td>Operational GovOps, on behalf of SPK Company Ltd, will transfer 6.5 billion SPK tokens from SPK Company Ltd account to an account specified by Sky.</td>
+          <td><p>SPK Company Ltd transferred 6.5 billion SPK tokens from the SPK Company Ltd account to the Sky Pause Proxy. The SPK Company Ltd account is specified in Minting Of Tokens To SPK Company Ltd.</p></td>
```

Comment:
nit: You don't need p tags for a single line entry

```suggestion
          <td>SPK Company Ltd transferred 6.5 billion SPK tokens from the SPK Company Ltd account to the Sky Pause Proxy. The SPK Company Ltd account is specified in <dfn>Minting Of Tokens To SPK Company Ltd</dfn>.</td>
```